### PR TITLE
Update contractor member policies

### PR DESCRIPTION
**Pull request by aesharpe**
### Describe the Policy Change
Not an official policy change, just clarified the distinction between contractor and member policies and added some links

### Reasoning and Concerns
- The contractor pay still said the same hourly wage as the employees which is not true. I updated this to reflect the policy that pays contractors more to account for the payroll taxes they have to pay.
- The tech stipend was offered to contractors in the member vs. contractor table when in reality it's only for members.
- There was no mention of the IRL stipend in the member vs. contractor table, so I added it.
- There were no links between the policies/meetings listed in the member vs. contractor tables and the policies and meetings descriptions below, so I added them. 
- Still some references to Steve that I updated / removed.
- Link to the generic meeting agenda template was bad so I fixed it.

### CODE REVIEW

**Review by jdangerx — APPROVED** [review]
👓

**Review by bendnorman — APPROVED** [review]
The docs failed because the [cross-referencing format was incorrect](https://myst-parser.readthedocs.io/en/latest/intro.html#cross-referencing).

**Review by jdangerx — APPROVED** [review]
Sweet, glad you figured it out! Is all that trailing whitespace in `catalyst_101` necessary too?

**Comment by bendnorman** [comment]
The whitespace after the `(header-name)=` tags?

File: docs/catalyst_101.md [hunk 1]
```diff
@@ -73,14 +73,14 @@ After taxes, contractors and members receive the same hourly rate. Employee-memb
...
+|                      | **Contractors**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | **Employee-Members**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (jdangerx) [inline docs/catalyst_101.md:76]: @bendnorman This is the whitespace I was talking about.

Comment (jdangerx) [reply docs/catalyst_101.md:76]: Or at least it showed up as whitespace in the split diff view 🤷

Comment (bendnorman) [reply docs/catalyst_101.md:76]: This is the format the markdown pre-commit hook came up with. Markdown tables are pretty messy.

**Review by bendnorman — APPROVED** [review]

**Review by jdangerx — APPROVED** [review]

**Pull request merged**